examples/01-Getting-started.ipynb

```diff
@@ -71,9 +71,11 @@
      "name": "stderr",
```

Comment:
<p>if you add <code>train.schema = train.schema.without(['rating'])</code> this will remove the rating column from the train.schema so that output block wont be confused of  two targets tagged as binary classification and regression, and the acc metrics will be printed properly. it wont print regression (hopefully).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/772/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>